src/components/popover/popover.directive.ts

```diff
@@ -9,7 +9,11 @@ import { PopoverComponent } from './popover.component';
 
 @Directive({
     selector: '[uxPopover]',
-    exportAs: 'ux-popover'
+    exportAs: 'ux-popover',
+    host: {
+        '[class.ux-popover-host]': 'true',
+        '[class.ux-tooltip-host]': 'false'
```

Comment:
Do uxPopover and uxTooltip need to have different host classes? They're both essentially the same thing, you can probably use a common class for both.

Reply:
No, I just did it this way so if you're inspecting it you won't see a `ux-tooltip-host` on a popover to avoid confusion. But I could switch it back to using `ux-tooltip-host` on both if you think that's simpler? I suppose it wouldn't matter too much since popover extends tooltip

Reply:
Yes I think that's fine, or if you have any ideas for a neutral name. It's not a big deal, just saves a couple of lines of code.